vignettes/finalize.Rmd

```diff
@@ -15,7 +15,8 @@ knitr::opts_chunk$set(
   fig.asp = 400/670,
   fig.retina = 4,
   fig.align = "center",
-  out.width = "100%"
+  out.width = "100%",
+  dev = "ragg_png"
```

Comment:
In finalize.Rmd there are three references to `cmapplot_globals` that need to be updated because that is no longer an exported object. They should probably be updated to `get_cmapplot_globals()` or `set_cmapplot_global()`.

Reply:
OK, I'm adjusting these.